dev/PowerNotifications/PowerNotifications.idl

```diff
@@ -3,7 +3,11 @@
 
 namespace Microsoft.Windows.System.Power
 {
+    [contractversion(1.1)]
```

Comment:
No MAJOR.MINOR.  Only 1 number.  Start with 1.  Any changes to the interface will increment the number.

Reply:
Where is that requirement coming from? Is that an API review guideline?

Reply:
@DrusTheAxe, @BenJKuhn, @DefaultRyan, and I came up with it.

Here is the spec currently for contracts: https://github.com/microsoft/WindowsAppSDK/pull/2110

It isn't an API review guideline.  The version gets messy when upgrading the MAJOR version that includes a patch.  IF you believe that having a MAJOR.MINOR version is better can you please add the comment to the PR I mentioned?